app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/NewTransferNavHost.kt

```diff
@@ -112,5 +119,15 @@ fun NewTransferNavHost(
                 },
             )
         }
+        composable<NewTransferFilesDetailsDestination> {
+            val backStackEntry = remember(it) { navController.getBackStackEntry(ImportFilesDestination) }
+            NewTransferFilesDetailsScreen(
+                importFilesViewModel = hiltViewModel<ImportFilesViewModel>(backStackEntry),
```

Comment:
Same question, does it make sense to keep the default `importFilesViewModel` value inside the method declaration?